src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0, // result is a property of the calculator object
+
+    // Define the basic operations
+    add: function (num) {
+      this.result += num;
+    },
+    subtract: function (num) {
+      this.result -= num;
+    },
```

Comment:
[CODE KNOWLEDGE] The same applies for the `subtract` method. Use the shorter syntax `subtract(num) {...}` instead of `subtract: function (num) {...}`.